ehcache-core/src/main/java/org/ehcache/core/spi/store/tiering/CachingTier.java

```diff
@@ -107,6 +108,22 @@ public interface CachingTier<K, V> extends ConfigurationChangeSupport {
    */
   void setInvalidationListener(InvalidationListener<K, V> invalidationListener);
 
+  /**
+   * Bulk method to get its mapped value from CachingTier, then look at AuthoritativeTier for the missing entries Of CachingTier then Compute a value for every key passed in the {@link Set} <code>keys</code> argument using the <code>mappingFunction</code>
```

Comment:
If you can rewrite this javadoc without referencing the `AuthoritativeTier` that would be good. Just describe what the method does... the authority only comes in with the functions we normally pass to this method.

Reply:
modified it, now not mentioning `AuthoritativeTier `